scripts/build-examples-mcp-resources.js

```diff
@@ -8,8 +8,41 @@
 const fs = require('fs');
```

Comment:
I'll probably do this next time I need to touch this file, but this file has really grown larger than I'd like and does many many things. ;) 

Time to 🪓 ?